externs/es3.js

```diff
@@ -935,6 +935,14 @@ Boolean.prototype.toSource = function() {};
 Boolean.prototype.toString = function() {};
 
 /**
+ * @return {boolean}
+ * @nosideeffects
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/valueOf
+ * @override
```

Comment:
I know but `@override` is present in other Boolean methods so I've just copied it. Do you want to me to remove it? If yes, from all of them?